src/tests/conformance/cmd/mod.rs

```diff
@@ -1,3 +1,4 @@
+mod endpoints;
```

Comment:
I'm wondering if these messages belong to the command module.
Maybe a `broadcast` module would describe what we are testing here since `TmManifsets` and `TmEndpoints` are messages the node constantly broadcasts to us without any query. And `TmValidator` messages, I guess.

Reply:
Yes, I didn't like it either. Since we're testing what happens after connection/startup, I think any name like `connection`, `protocol`, `startup` will be more-or-less ok. We can move all tests like this in another PR once we decide how to call them.

Reply:
> We can move all tests like this in another PR once we decide how to call them.

Yes, sure!

But my view on the directory categories is based on the nature of the messages:
- When we send a query and expect a response - `query` mod (e.g. ping)
- When we send an order/command to a node (and not expect a direct answer) - `cmd` mod (e.g. squelch)
- When we do nothing but still receive some messages from a node - `maybe` broadcast mod (e.g. `TmValidation`)

Reply:
Agreed. Let's do in the next PR.